src/containers/User/UserArea.tsx

```diff
@@ -54,16 +56,42 @@ const MainMenuBar: React.FC = () => {
   )
 }
 
-const OrgSelector: React.FC<{ user: User }> = ({ user }) => {
-  // TO-DO: Make into Dropdown so Org can be selected
+const OrgSelector: React.FC<{ user: User; orgs: OrganisationSimple[]; onLogin: Function }> = ({
+  user,
+  orgs,
+  onLogin,
+}) => {
+  const LOGIN_AS_NO_ORG = 0 // Ensures server returns no organisation
```

Comment:
not required change, and not part of this PR, but I like the idea of using context state vs passing through params (`useUserState` here i mean)